packages/react-components/react-overflow/src/useOverflowContainer.ts

```diff
@@ -25,7 +27,14 @@ export const useOverflowContainer = <TElement extends HTMLElement>(
   update: OnUpdateOverflow,
   options: Omit<ObserveOptions, 'onUpdateOverflow'>,
 ): UseOverflowContainerReturn<TElement> => {
-  const { overflowAxis, overflowDirection, padding, minimumVisible, onUpdateItemVisibility } = options;
+  const {
+    overflowAxis,
+    overflowDirection,
+    padding,
+    minimumVisible,
+    onUpdateItemVisibility,
```

Comment:
probable worth deprecating `onUpdateItemVisibility` and have something more generic and extendable like `onUpdateOverflowState`

let's not have a separate `onUpdateDividerVisibility`

Reply:
yeah, it was an experiment with hiding dividers in overflow

Reply:
resolving, as this thing is removed